README.md

```diff
@@ -22,6 +22,58 @@ npm i -S react-native-component-viewer
 
 This library does not assume any specific navigation library is in use. As a result it can be configured for us with [react-navigation](https://github.com/react-community/react-navigation), [react-native-router-flux](https://github.com/aksonov/react-native-router-flux), and others.
 
+## Usage in react-navigation
+
+Here's how to use it in react-native-router-flux:
```

Comment:
Copy paste error here - this should say `in react-navigation`. Gonna just fix it.